### PR TITLE
feat: add reverse scans

### PR DESCRIPTION
**Pull request by brandtnewton**

### CODE REVIEW

File: cbt.go [hunk 1]
```diff
@@ -1409,7 +1410,7 @@ func doRead(ctx context.Context, args ...string) {
 	parsed, err := parseArgs(args[1:], []string{
 		"start", "end", "prefix", "columns", "count",
 		"cells-per-column", "regex", "app-profile", "limit",
-		"format-file", "keys-only", "include-stats",
+		"format-file", "keys-only", "include-stats", "reversed",
```
Comment (brandtnewton) [inline cbt.go:1413]: "reversed" or "reverse"?

File: cbt.go [hunk 2]
```diff
@@ -1422,11 +1423,20 @@ func doRead(ctx context.Context, args ...string) {
...
-	} else if start != "" {
-		rr = bigtable.InfiniteRange(start)
+	start, end := parsed["start"], parsed["end"]
+	if reversed {
+		rr = bigtable.NewOpenClosedRange(start, end)
+	} else {
+		rr = bigtable.NewClosedOpenRange(start, end)
 	}
 	if prefix := parsed["prefix"]; prefix != "" {
 		rr = bigtable.PrefixRange(prefix)
```
Comment (brandtnewton) [inline cbt.go:1442]: considering the Open-Closed/Closed-Open changes above, should the prefix have a similar behavioral change depending on the "reversed" flag?

File: cbt.go (hunk 1 above)
Comment (brandtnewton) [reply cbt.go:1413]: "reversed": https://cloud.google.com/bigtable/docs/reads

File: cbtdoc.go [hunk 3]
```diff
@@ -273,7 +273,7 @@ Usage:
 
 	    Example: cbt createinstance my-instance "My instance" my-instance-c1 us-central1-b 3 SSD
 
-# Create a backup from a source table (deprecated)
```
Comment (brandtnewton) [inline cbtdoc.go:276]: why is this being changed in the docs? Seems to only effect command descriptions that contain `(deprecated)`

File: cbt.go [hunk 4]
```diff
@@ -1422,11 +1423,20 @@ func doRead(ctx context.Context, args ...string) {
...
+	}
+
 	var rr bigtable.RowRange
-	if start, end := parsed["start"], parsed["end"]; end != "" {
-		rr = bigtable.NewRange(start, end)
-	} else if start != "" {
-		rr = bigtable.InfiniteRange(start)
+	start, end := parsed["start"], parsed["end"]
+	if reversed {
+		rr = bigtable.NewOpenClosedRange(start, end)
```
[the remark this answers is not in the document]
Comment (brandtnewton) [reply cbt.go]: more ergonomic but might be unexpected behavior for users so not doing it

File: cbt.go (hunk 2 above)
Comment (brandtnewton) [reply cbt.go]: n/a

**Review by igorbernstein2 — APPROVED** [review]

**Pull request merged**